src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -611,6 +620,22 @@ public static void GetFullPath_Windows_ValidExtendedPaths(string path)
             }
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific paths
+        [Theory]
+        [InlineData(@"\\.\UNC\")]
+        [InlineData(@"\\.\UNC\server7")]
```

Comment:
I just copied this InlineData from the previous test as I partitioned them into two different tests.
https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/IO/PathTests.cs#L584

I can fix it and update them to be localhost though. Should I?